### PR TITLE
CXP-1369: DSM - Add a size property to the boolean input

### PR DESCRIPTION
**Pull request by eoliveauakeneo**
**Description (for Contributor and Core Developer)**

- New property "size" for the boolean input which take "normal" (default) or "small"- 

image

### CODE REVIEW

File: front-packages/akeneo-design-system/src/components/Input/BooleanInput/BooleanInput.tsx [hunk 1]
```diff
@@ -12,13 +12,14 @@ const BooleanButton = styled.button<
...
   } & AkeneoThemedProps
 >`
   ${CommonStyle}
-  height: 40px;
-  width: 60px;
+  height: ${({size}) => ('normal' === size ? 40 : 30)}px;
+  width: ${({size}) => ('normal' === size ? 60 : 48)}px;
   display: inline-block;
-  line-height: 36px;
+  line-height: ${({size}) => ('normal' === size ? 36 : 26)}px;
```
[the remark this answers is not in the document]
Comment (eoliveauakeneo) [reply front-packages/akeneo-design-system/src/components/Input/BooleanInput/BooleanInput.tsx:22]: You're right, i will do that

**Review by RiffVentura — APPROVED** [review]

File: front-packages/akeneo-design-system/src/components/Input/BooleanInput/BooleanInput.stories.mdx [hunk 2]
```diff
@@ -119,3 +122,25 @@ The input can in some cases have a 3rd state, EMPTY. In this case, a clear butto
     }}
   </Story>
 </Canvas>
+
+## Variation on small
```
Comment (masacc) [inline front-packages/akeneo-design-system/src/components/Input/BooleanInput/BooleanInput.stories.mdx:126]:
```suggestion
## Variation on size
```

File: front-packages/akeneo-design-system/src/components/Input/BooleanInput/BooleanInput.stories.mdx [hunk 3]
```diff
@@ -34,6 +34,9 @@ import {useArgs, updateArgs} from '@storybook/client-api';
     clearLabel: {
       description: 'The label for the Clear value button',
     },
+    size: {
+      description: 'The size of the buttons',
+    },
```
Comment (masacc) [inline front-packages/akeneo-design-system/src/components/Input/BooleanInput/BooleanInput.stories.mdx:39]:
```suggestion
    size: {
      description: 'The size of the buttons',      
      table: {
        defaultValue: {summary: 'normal'}
      },      
      control: {type: 'select'}, options: ['normal', 'small']
    },
```
test if it works but it should allow seeing the default value and a select to live-update the playground canva

**Review by masacc — APPROVED** [review]

File: front-packages/akeneo-design-system/src/components/Input/BooleanInput/BooleanInput.stories.mdx (hunk 3 above)
Comment (eoliveauakeneo) [reply front-packages/akeneo-design-system/src/components/Input/BooleanInput/BooleanInput.stories.mdx:39]: Yep 
image

**Review by guillaumefer-akeneo — APPROVED** [review]

**Pull request merged**